src/libraries/System.Security.Cryptography/tests/X509Certificates/ChainTests.cs

```diff
@@ -1178,12 +1178,12 @@ public static void BuildChainForCertificateSignedWithDisallowedKey()
                 chain.ChainPolicy.ExtraStore.Add(intermediateCert);
                 Assert.False(chain.Build(cert));
 
-                if (PlatformDetection.IsAndroid)
+                if (PlatformDetection.IsAndroid || PlatformDetection.IsApplePlatform26OrLater)
                 {
                     // Android always validates trust as part of building a path,
                     // so violations comes back as PartialChain with no elements
+                    // Apple 26 no longer block these SKIs since the roots are no longer trusted at all and are expired.
                     Assert.Equal(X509ChainStatusFlags.PartialChain, chain.AllStatusFlags());
```

Comment:
Perhaps we should remove the Skip(Linux) and just add Linux to the "what explicit distrust?" party